addons/payment/controllers/portal.py

```diff
@@ -355,12 +354,12 @@ def _update_landing_route(tx_sudo, access_token):
                                 f'?tx_id={tx_sudo.id}&access_token={access_token}'
 
     @http.route('/payment/confirmation', type='http', methods=['GET'], auth='public', website=True)
-    def payment_confirm(self, tx_id, access_token, **kwargs):
+    def payment_confirm(self, tx_id, access_token, **_kwargs):
         """ Display the payment confirmation page to the user.
 
         :param str tx_id: The transaction to confirm, as a `payment.transaction` id
         :param str access_token: The access token used to verify the user
-        :param dict kwargs: Optional data. This parameter is not used here
+        :param dict _kwargs: Optional data. This parameter is not used here
         :raise: werkzeug.exceptions.NotFound if the access token is invalid
```

Comment:
Can't comment the python line so commenting the comment line but missing `()`, raise NotFound**()**

Reply:
Turns out it works just fine like this :o Let's fix it with this PR and in master only then, for the sake of doing it correctly.

Reply:
Yes, but you don't take the shorter path with it.
Since you will be handled by handle_exception and others codes without an instanceOf the exception. Luckily using the exception code, he will raise what you want later...

Reply:
about the _kwargs, it just that it is a standard to use kwargs / kw

So rename it freely just for your linter is a strange reason imo.

It is like if I decide to rename tomorrow in a subset of module the _logger by _logging...
It is not wrong but make no plusvalue, except to increase the risk of error, and the need to double check each time the logger name...